lib/datadog/open_feature/transport/http/exposures.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require_relative '../../../core/transport/http/api/endpoint'
+require_relative '../../../core/transport/http/api/instance'
+require_relative '../../../core/transport/http/api/spec'
+
+require_relative 'client'
+
+module Datadog
+  module OpenFeature
+    module Transport
+      module HTTP
+        # HTTP transport behavior for exposure events
+        module Exposures
+          # Extensions for HTTP client
+          module Client
+            def send_exposures_payload(request)
+              send_request(request) do |api, env|
+                api.send_exposures(env)
+              end
+            end
+          end
+
+          module API
+            # HTTP API Spec
+            class Spec < Core::Transport::HTTP::API::Spec
+              attr_accessor :endpoint
+
+              def send_exposures(env, &block)
+                if endpoint.nil?
+                  raise Core::Transport::HTTP::API::Spec::EndpointNotDefinedError.new(
+                    'exposures', self
+                  )
+                end
+
+                endpoint.call(env, &block)
+              end
+            end
+
+            # HTTP API Instance
+            class Instance < Core::Transport::HTTP::API::Instance
+              def send_exposures(env)
+                unless spec.is_a?(Exposures::API::Spec)
+                  raise Core::Transport::HTTP::API::Instance::EndpointNotSupportedError.new(
+                    'exposures', self
+                  )
+                end
+
+                spec.send_exposures(env) do |request_env|
+                  call(request_env)
+                end
+              end
+            end
+
+            # Endpoint for submitting exposure events data
+            class Endpoint < Datadog::Core::Transport::HTTP::API::Endpoint
+              HEADER_CONTENT_TYPE = 'Content-Type'
+              HEADER_SUBDOMAIN = 'X-Datadog-EVP-Subdomain'
+              SUBDOMAIN_VALUE = 'event-platform-intake'
+
+              attr_reader :encoder
+
+              def initialize(path, encoder)
+                super(:post, path)
+                @encoder = encoder
+              end
```

Comment:
Minor: Since we always use the same encoder and path, consider hardcoding both here (instead of in `HTTP::API`. All of this complexity is leftover from multiple APIs and versions in tracing, but I think it's not worth paying the complexity cost until we find that we need it (and even for tracing in many cases it was never used...)